cmd/zar/zq/command.go

```diff
@@ -6,28 +6,26 @@ import (
 	"os"
 
 	"github.com/brimsec/zq/archive"
-	"github.com/brimsec/zq/ast"
 	"github.com/brimsec/zq/cmd/zar/root"
 	"github.com/brimsec/zq/driver"
 	"github.com/brimsec/zq/emitter"
-	"github.com/brimsec/zq/pkg/iosrc"
 	"github.com/brimsec/zq/pkg/signalctx"
-	"github.com/brimsec/zq/zbuf"
 	"github.com/brimsec/zq/zio"
-	"github.com/brimsec/zq/zio/detector"
 	"github.com/brimsec/zq/zng/resolver"
 	"github.com/brimsec/zq/zql"
 	"github.com/mccanne/charm"
 )
 
 var Zq = &charm.Spec{
 	Name:  "zq",
-	Usage: "zq [-R root] [options] [zql] file [file...]",
-	Short: "walk an archive and run zql queries",
+	Usage: "zq [-R root] [options] zql [file...]",
+	Short: "execute ZQL against all archive directories",
 	Long: `
-"zar zq" executes a ZQL query against each chunk or associated file in an 
-archive. The special name "_" refers to chunk file itelf, and other names
-are interpreted relative to the chunk's associated file directory.
+"zar zq" executes a ZQL query against one or more files from all the directories
```

Comment:
Not for now, as this isn't new to this PR, but reading this help text makes me think we'll want to document these sub-commands in more user-visible terms, without it being a description of mechanics invoking the directory layout etc. Easier said than done but we'll get there 🙂. cc @philrz.